packages/web/components/LoginButton.tsx

```diff
@@ -45,7 +46,9 @@ export const LoginButton: React.FC = () => {
             {user.player ? getPlayerName(user.player) : 'Unknown'}
           </MetaLink>
           <HStack spacing={2}>
-            <MetaLink href="/profile/setup">Setup profile</MetaLink>
+            <MetaLink href="/profile/setup/username">
+              {user.username && user.username !== address ? 'Edit profile' : 'Setup profile'}
```

Comment:
I'd use `user?.username` since `user` can be `null`.

Reply:
no, there's a if `(!user?.player) return null;`above it, and a nullreference error would occur before here anyway.

Reply:
The type checker would've picked it up anyways if user could've been null here